CHANGES.rst

```diff
@@ -130,6 +130,10 @@ srctype
 
 - Updated logic for background targets and nodded exposures. [#3310]
 
+tests
+-----
+- Removed set do_copy = True for get_data [#3370] 
```

Comment:
I'm not sure we really need a change log entry for this, but if we do want it there, then the description needs to be expanded to indicate which test(s) we're talking about.

Reply:
In the past, we have not added any entries to CHANGES.rst for test infrastructure.  I suspect we don't need to, as it is internal to the package and not user-facing.

Reply:
I was not sure what to do about changing the base class in the tests directory.
It seems I should log this but I was not sure were. I make it clearer what I did - changed
the base class.